styles.css

```diff
@@ -487,7 +487,8 @@ hr {
   }
 
   .more {
-    display: none;
+    display: flex;
+    width: 40%;
```

Comment:
- [ ] please note that the view more button should be only visible in **_mobile view_** , the desktop should display  two columns of three articles each(6 total) by default .

this is how it should look : 

Desktop           | Mobile
:-------------------------:|:-------------------------:
![image](https://github.com/Surafels/Capstone_Project/assets/89282221/34fb3bd6-db85-4768-81dc-506a9b70bf9e)| ![image](https://github.com/Surafels/Capstone_Project/assets/89282221/07c216e6-c0e9-4ba9-a8a2-71f43ee3676a)



kindly remove the view more btn from the desktop view so it shows all the 6 articles by default 🙏 


Incorrect  ❌         | Correct ✔️ 
:-------------------------:|:-------------------------:
![image](https://github.com/Surafels/Capstone_Project/assets/89282221/50f57fd4-e1d2-4ba0-bc99-07c793764baf) | ![image](https://github.com/Surafels/Capstone_Project/assets/89282221/34fb3bd6-db85-4768-81dc-506a9b70bf9e)|